custom_components/deebot/translations/it.json

```
@@ -2,20 +2,37 @@
   "config": {
     "abort": {
       "already_configured": "Già configurato"
```

```suggestion
      "already_configured": "Già configurato",
```

Please verify that the json is valid